examples/lc709203f/default/main/Kconfig.projbuild

```diff
@@ -0,0 +1,36 @@
+menu "Example configuration"
+    config EXAMPLE_I2C_MASTER_SCL
+        int "SCL GPIO Number"
+        default 5 if IDF_TARGET_ESP8266
+        default 6 if IDF_TARGET_ESP32C3
+        default 19 if IDF_TARGET_ESP32 || IDF_TARGET_ESP32S2 || IDF_TARGET_ESP32S3
+        help
+            GPIO number for I2C Master clock line.
+
+    config EXAMPLE_I2C_MASTER_SDA
+        int "SDA GPIO Number"
+        default 4 if IDF_TARGET_ESP8266
+        default 5 if IDF_TARGET_ESP32C3
+        default 18 if IDF_TARGET_ESP32 || IDF_TARGET_ESP32S2 || IDF_TARGET_ESP32S3
+        help
+            GPIO number for I2C Master data line.
+    
+    choice EXAMPLE_BOARD
+        prompt "Example Board"
```

Comment:
use `bool "Set HIGH on extra GPIO to enable onboard pullup resistors"`, which describes exactly what the option does.

Reply:
i don't know this is a common practice on Adafruit boards. but if it is, there should be an additional option to choose the GPIO number.

Reply:
It's my first board from Adafruit. I neither know if it's a common practice. I only know that esp32s2/s3 feather versions have it.

I feel this is too much board specific option and must be deleted. User is free to adapt example code for board used.

If you agree, I will delete this code.

Wait your response.

Reply:
examples are not only for users, but also for developers. it's okay to have specific quirks in examples. the quirk just needs to be clear enough for users that it is for some specific boards.

Reply:
I have changed configuration again. Instead select a board, I have added new option to select GPIO number to configure as output or -1 if no needed. If pin number is >=0, new option to set pin level is activated.

I think it is easier to undertand. Review it.